protocol/x/clob/keeper/process_operations_liquidations.go

```diff
@@ -11,6 +11,56 @@ import (
 	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
 )
 
+// ValidateLiquidationOrderAgainstProposedMatch performs stateless validation of a liquidation order
+// against a proposed match.
```

Comment:
Small suggestion: I think this function name is slightly clearer (when I first read this it sounded like the validation was being performed on the maker fills):

```suggestion
// ValidateLiquidationOrderAgainstProposedLiquidation performs stateless validation of a liquidation order
// against a proposed liquidation.
```